### ISSUE TITLE
hyprland-workspaces crashes on launch

### ISSUE THREAD

**Issue by varris1** [FieldofClay/hyprland-workspaces#19]
hyprland-workspaces crashes for me on launch because of the following error:
```thread 'main' panicked at /build/hyprland-workspaces-1.2.5-vendor.tar.gz/hyprland/src/data/regular.rs:31:19:
A error occured while parsing the output from the hypr socket: IoError(Os { code: 2, kind: NotFound, message: "No such file or directory" })
stack backtrace:
   0: rust_begin_unwind
   1: core::panicking::panic_fmt
   2: hyprland::data::regular::call_hyprctl_data_cmd
   3: <hyprland::data::regular::Monitors as hyprland::shared::HyprData>::get
   4: hyprland_workspaces::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

I assume it's related to the changes in hyprland socket location. It was moved from /tmp/hypr to $XDG_RUNTIME_DIR/hypr.

hyprland-activewindow is also affected by this.

EDIT: It was fixed in hyprland-rs, but you may need to bump the hyprland crate to a git version until it they release a new version

**Comment by FieldofClay** [comment]
Yeah, it is hyprland-rs that fixes this. I've released [2.0.1-beta](https://github.com/FieldofClay/hyprland-workspaces/releases/tag/v2.0.1-beta) which uses the latest hyprland-rs alpha version that contains the fix. I'll do a release for hyprland-activewindow shortly as well.

Given there are more changes than just the socket fix, I'll wait until 0.4.0 is out fully before releasing a stable.

**Comment by FieldofClay** [comment]
Bumped version to stable, closing.
